jenkins/spark-nightly-build.sh

```diff
@@ -21,36 +21,80 @@ set -ex
 
 ## export 'M2DIR' so that shims can get the correct cudf/spark dependency info
 export M2DIR="$WORKSPACE/.m2"
+
+DIST_PL="dist"
+function mvnEval {
+    mvn help:evaluate -q -pl $DIST_PL $MVN_URM_MIRROR -Dmaven.repo.local=$M2DIR -Dcuda.version=$CUDA_CLASSIFIER -DforceStdout -Dexpression=$1
+}
+
+ART_ID=$(mvnEval project.artifactId)
+ART_GROUP_ID=$(mvnEval project.groupId)
+ART_VER=$(mvnEval project.version)
+
+DIST_FPATH="$DIST_PL/target/$ART_ID-$ART_VER"
+DIST_POM_FPATH="$DIST_PL/target/extra-resources/META-INF/maven/com.nvidia/$ART_ID/pom.xml"
```

Comment:
nit could use ART_GROUP_ID for com.nvidia

Reply:
DIST_POM_FPATH is never used

Reply:
will remove, I first wanted to use it as for releases but then remembered that #4265  fixed META-INF/maven pom entry inside the jar to be consistent with the published pom